spec.go

```diff
@@ -134,12 +134,12 @@ func Analyzed(data json.RawMessage, version string) (*Document, error) {
 }
 
 // Expanded expands the ref fields in the spec document and returns a new spec document
-func (d *Document) Expanded() (*Document, error) {
+func (d *Document) Expanded(options *spec.ExpandOptions) (*Document, error) {
```

Comment:
Would it be possible to make this ExpandedWithOptions and keep the Expanded method without arguments.
Then the expand options could have as default option the dir of the path of the spec it loaded